third-party/qthread/qthread-src/config/qthread_check_hwloc.m4

```diff
@@ -30,7 +30,7 @@ AC_DEFUN([QTHREAD_CHECK_HWLOC], [
   AC_CHECK_HEADERS([hwloc.h],[],
   				   [qt_allgoodsofar=no])
   AS_IF([test "x$qt_allgoodsofar" = xyes],
-	    [AC_SEARCH_LIBS([${with_hwloc_symbol_prefix}topology_init], [hwloc "hwloc -lnuma"], [],
+	    [AC_SEARCH_LIBS([${with_hwloc_symbol_prefix}topology_init], [hwloc "hwloc -lnuma" "hwloc -lnuma -lpciaccess"], [],
```

Comment:
Should also add `"hwloc -lpciaccess"` for systems with libpciaccess but not libnuma.

Reply:
Done.